composer.json

```diff
@@ -8,7 +8,16 @@
   "authors" : [ {
     "name" : "Wim Godden",
     "role" : "lead"
-  } ],
+  },
+  {
+    "name" : "Juliette Reinders Folmer",
+    "role" : "lead"
+  },
+{
+    "name" : "https://github.com/PHPCompatibility/PHPCompatibility/graphs/contributors",
+    "role" : "contributors"
+}
+    ],
```

Comment:
Either have all of the sub-sets indented and the block opener/closer on their own line or have the closer of the last sub-set on the same line as the closer of the block.
Currently, the block opener + opener of the first sub-set are on the same line, but for the closer, they are on different lines.